library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterInputTest.java

```diff
@@ -155,6 +155,38 @@ public void testFileMetadata() throws InterruptedException
   }
 
   @Test
+  public void testFileMetadataForSingleFile() throws InterruptedException
```

Comment:
there is already a test with name "testSingleFile" can we rename this to something like, "testScannerFilterForDuplicates"